grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx

```diff
@@ -106,11 +132,38 @@ const ManualAlertGroup: FC<ManualAlertGroupProps> = (props) => {
               <ul className={cx('responders-list')}>
                 <li>
                   <HorizontalGroup justify="space-between">
-                    <Text>{selectedTeamDirectPaging.verbal_name}</Text>
+                    <HorizontalGroup>
+                      {integrationDoesNotHaveEscalationChains && (
+                        <Tooltip content="Integration doesn't have connected escalation policies">
+                          <Icon name="exclamation-triangle" style={{ color: 'var(--warning-text-color)' }} />
+                        </Tooltip>
+                      )}
+                      <Text>{selectedTeamDirectPaging.verbal_name}</Text>
+                    </HorizontalGroup>
                     <HorizontalGroup>
                       <Text type="secondary">Team:</Text>
                       <TeamName team={store.grafanaTeamStore.items[selectedTeamId]} />
                     </HorizontalGroup>
+                    <HorizontalGroup>
+                      {chatOpsAvailableChannels && (
+                        <>
+                          <Text type="secondary">ChatOps:</Text>{' '}
+                          {chatOpsAvailableChannels.map(
+                            (chatOpsChannel: { name: string; icon: IconName }, chatOpsIndex) => (
+                              <div
+                                key={`${chatOpsChannel?.name}-${chatOpsIndex}`}
```

Comment:
maybe `key=chatOpsChannel?.name+chatOpsChannel?.icon`? and actually you can get rid of  question mark here `chatOpsChannel?.name` because you iterate over array and chatOpsChannel cannot be undefined inside loop